filters/filter.go

```diff
@@ -53,9 +53,10 @@
 // labels."very complex label"==something
 // ```
 //
-// We also define `!=` and `~=` as operators. The `!=` will match all
-// objects that don't match the value for a field and `~=` will compile the
+// We also define `!=`, `~=`, `@=` as operators. The `!=` will match all
+// objects that don't match the value for a field. `~=` will compile the
 // target value as a regular expression and match the field value against that.
+// `@=` is a "contains" substring operator.
```

Comment:
Isn't regexp enough to match "contains" ?

Reply:
That works too, reverted/updated.